internal/controllers/post_controller.go

```diff
@@ -188,3 +190,32 @@ func (controller *PostController) GetPostFeed(c *gin.Context) {
 	}
 	c.JSON(http.StatusOK, postFeed)
 }
+
+func (controller *PostController) DeletePost(c *gin.Context) {
+	postIdStr := c.Param("postId")
+	postId, err := uuid.Parse(postIdStr)
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"error": customerrors.BadRequest,
+		})
+		return
+	}
```

Comment:
Wir haben ja heute über die Verwendung von Bad Request gesprochen. BadRequest senden wir jetzt nur noch wenn wirklich der Body falsch ist.
Macht es wie folgt: lest einfach die PostID als String aus ohne sie zu UUID zu konvertieren und übergebt sie als String an den Service. Falls die PostID dann leer ist/kein zugehöriger Post exsitiert/kein UUID Format hat/etc., dann kommt sowieso ein 404 Not Found